daemon/container_linux.go

```diff
@@ -999,7 +1005,7 @@ func (container *Container) getNetworkedContainer() (*Container, error) {
 }
 
 func (container *Container) ReleaseNetwork() {
-	if container.hostConfig.NetworkMode.IsContainer() || container.daemon.config.DisableNetwork {
+	if container.hostConfig.NetworkMode.IsContainer() {
```

Comment:
@mrjana the reason I removed this is that, in case of NetworkDisabled, the eid and nid are "" and the subsequent checks takes care of it. But I can add this back.